src/NDItk/multigroup/OutgoingParticleTypes/src/verify.hpp

```diff
@@ -0,0 +1,18 @@
+/**
+ *  @brief Verify the weights
+ *
+ *  The following verification tests are performed:
+ *    - there is at least one value
+ *
+ *  @param[in] values    the velocity values to be verified
+ */
+template < typename Range >
+static void verify( const Range& values ) {
+
+  if ( values.size() < 1 ) {
```

Comment:
I have another "verify" check in mind for several objects that we can add later. Example: make sure none of the values are < 0 (any time there's an integer zaid) and make sure atomic weights and atomic weight rations are >= 0. For the future.

Reply:
Yes, we should make a note on tests that we want to perform.